tasks/steps/wait_members_alive.yml

```diff
@@ -18,6 +18,5 @@
       until: not check_members_alive_res.failed
       retries: '{{ wait_members_alive_retries }}'
       delay: '{{ wait_members_alive_delay }}'
-      when: not expelled
       run_once: true
```

Comment:
In fact, it's a bugfix that should be committed separately =)

Reply:
But in this PR we make the disabled flags near by expelled flags, so we can say that this is part of the PR :)